scarb/tests/package.rs

```diff
@@ -63,3 +177,59 @@ fn list_workspace() {
             src/lib.cairo
         "#});
 }
+
+#[test]
+fn reserved_files_collision() {
+    let t = TempDir::new().unwrap();
+    ProjectBuilder::start()
+        .name("foo")
+        .version("1.0.0")
+        .src("VERSION", "oops")
+        .src("Scarb.orig.toml", "oops")
+        .build(&t);
+
+    Scarb::quick_snapbox()
+        .arg("package")
+        .current_dir(&t)
+        .assert()
+        .failure()
+        .stdout_matches(formatdoc! {r#"
+        [..] Packaging foo v1.0.0 [..]
+        error: invalid inclusion of reserved files in package: VERSION, Scarb.orig.toml
+        "#});
+}
+
+// TODO(mkaput): Manifest normalization, esp in workspaces
+// TODO(mkaput): Git & local paths
+// TODO(mkaput): Symlinks and other FS shenanigans
+// TODO(mkaput): Gitignore
+// TODO(mkaput): Invalid readme/license path
+// TODO(mkaput): Restricted Windows files
+
+#[test]
+fn clean_tar_headers() {
+    let t = TempDir::new().unwrap();
+    ProjectBuilder::start()
+        .name("foo")
+        .version("1.0.0")
+        .lib_cairo("mod foo;")
+        .src("src/foo.cairo", "fn foo() {}")
+        .build(&t);
+
+    Scarb::quick_snapbox()
+        .arg("package")
+        .current_dir(&t)
+        .assert()
+        .success();
+
+    let mut archive = PackageChecker::open(&t.child("target/package/foo-1.0.0.tar.zst"));
+    for entry in archive.entries().expect("failed to get archive entries") {
+        let entry = entry.expect("failed to get archive entry");
+        println!("checking: {:?}", entry.path());
```

Comment:
Do we need this line?

Reply:
Removing it might make tests harder to debug, but indeed it interferes with tests output. Removing then.